build.sh

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/bash -eu
 
-set -eu
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
```

Comment:
I can see your argument.  I tend to pair it with a comment describing it is bash Parameter Expansion from the man page.